src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -391,7 +439,20 @@ private void OnUpdateGraphCompleted(AsyncTask task)
 
             OnEvaluationCompleted(e);
 
+            if (EngineController.IsDisposed) return;
+
             EngineController.ReconcileTraceDataAndNotify();
+
+            // Refresh values of nodes that took part in update.
+            foreach (var modifiedNode in updateTask.ModifiedNodes)
+            {
+                modifiedNode.RequestValueUpdateAsync(scheduler, EngineController);
+            }
+
+            scheduler.Tasks.AllComplete(_ =>
+            {
+                OnRefreshCompleted(e);
+            });
```

Comment:
I think it would help diagnostics to have some sense as to how many of these are piling up. I'm a bit concerned that we may accumulate them if something goes wrong that causes some of the tasks to not exit cleanly for some reason. I don't think there's much we can do to recover automatically, but it'd be nice if these were a little more debuggable